### DISCUSSION TITLE
My app build history dissappeared

### DISCUSSION THREAD

**Discussion by iVnews-Devops-Master** [codemagic-ci-cd/codemagic-docs#1336] [Q&A] [answered]
Hello,

I just checked in and was going to build a new version of my app but when I did I found that my connection to gitlab was no longer working and my build history was gone. I managed to restore the gitlab connection after disconnecting and reconnecting the team to it but the build history is still missing... help?

Kind Regards,
John

**Comment by K-Codemagic** [comment, marked as answer]
Hi, can you please share the app id or build id so I can look into it? However, the Build history and artifacts of apps on the personal account are stored in Codemagic for 30 days after which they are deleted.
